webui/react/src/hooks/usePermissions.test.tsx

```diff
@@ -32,7 +32,6 @@ const PermissionRenderer: React.FC<Props> = () => {
     canDeleteWorkspace,
     canModifyWorkspace,
     canViewWorkspace,
-    canViewWorkspaces,
```

Comment:
This implementation LGTM. Can we make sure to follow up on how we are going to implement `canViewWorkspaces`.